.github/contributing.md

```diff
@@ -1,9 +1,12 @@
 # Contributing
 
-## Security
+## Security / Disclosure
```

Comment:
I'm not sure if `docs:` or `chore:` is the best semantic commit scope.
I'll let you maintainers decide. 😉

Reply:
Either is fine, i would prefer docs